assets/queries/openAPI/schema_object_using_enum_with_keyword/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	keywords := ["multipleOf", "maximum", "minimum", "exclusiveMaximum", "exclusiveMinimum", "pattern", "minLength", "maxLength", "maxItems", "minItems", "uniqueItems", "items", "required", "maxProperties", "minProperties"]
+
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	[path, value] := walk(doc)
+	properties := value.properties[name]
```

Comment:
I think here should be `schema` and `schemas[name]`, since we want to check all schema objects and not only the properties of a schema typed as object...

Reply:
resolved

Reply:
I belive it should be properties since, `enum` can only be used inside schema.properties